inst/tutorials/r_and_rstudio_basic/r_and_rstudio_basic.Rmd

```diff
@@ -137,20 +135,6 @@ Let's create an R script (File > New File > R Script) and save it as `tidyverse.
 
 We can copy and paste the previous commands in this tutorial and aggregate it in our R script.
 
-```{r Scripts, echo=FALSE}
-quiz(
-  question("How do R scripts make your work reproducible?",
-    answer("Trick question: they work just like Excel and don't make work reproducible"),
-    answer("They keep a record of all actions done to get from raw data to final results", correct=TRUE),
-    answer("You can use them to test different operations on your data")
-),
```

Comment:
Is it worth introducing the `set.seed()` function for reproducible random simulations? Might be out of the scope of this tutorial but it's also a pretty straightforward function. What do you think @cathy-y and @stephan-koenig?